### DISCUSSION TITLE
How to maintain a group of shapes' placements during rotation by Transformer?

### DISCUSSION THREAD

**Discussion by lanyusan** [konvajs/konva#1230] [Q&A]
I am further building my prototype with Konva.

I have found the transformer very handy but there is a problem with rotation of a group of shapes.

I am not sure how the transformer pick the rotation center. The result is that the shapes' placements weren't kept exactly as they were related to each other before rotation.

Please find the images:

After:
![Screenshot from 2021-11-15 15-53-50](https://user-images.githubusercontent.com/56706512/141743674-c3dc8e4c-df11-4a96-8845-ccf73989732e.png)

Before:
![Screenshot from 2021-11-15 15-53-02](https://user-images.githubusercontent.com/56706512/141743706-7211f792-aba1-44ca-b96d-254ce11f39ed.png)

Is there a way to tune the transformer so when the group is rotated, all shapes maintain their relative locations? 
I suspect somehow during rotation the rotation center shifted.

**Comment by lavrton** [comment]
Make a demo.

**Reply by lanyusan** [reply]
This is tied to the prototype that has a in-house built drawing UI  but it is not allowed to share all the code.
I will provide a demo just for this when I am freed up. 
Thanks.
